plugins/snap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: snap
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/honk-ci/kubectl-snap
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/honk-ci/kubectl-snap/releases/download/0.1.0/kubectl-snap-0.1.0.tar.gz
+    sha256: "8df10f76e82f71b34c737d128b7b369e0daf641118577822872537e787a6755d"
+    bin: kubectl-snap
```

Comment:
You archive lists the following files:
```
.gitignore
LICENSE
README.md
hack/
hack/release.sh
kubectl-snap
```
Some of these files don't need to be installed on the users' machine. Can you add a `files` section to just install `LICENSE` and `kubectl-snap`?

Reply:
Will do.